### PR TITLE
Added changelog

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+  
+### Technical Improvements 
+
+- Check if legacy group already exists on send [\#269](https://github.com/PegaSysEng/orion/pull/269)
+- Updating CircleCI jobs to Java 11 [\#267](https://github.com/PegaSysEng/orion/pull/267) 
+- Null checks name and description of privacy group, and associated test [\#265](https://github.com/PegaSysEng/orion/pull/265) (thanks to [josh-richardson](https://github.com/josh-richardson))
+- Always return empty list when privacy group id doesn't exist [\#264](https://github.com/PegaSysEng/orion/pull/264) 
+- Adding error msg on /receive [\#263](https://github.com/PegaSysEng/orion/pull/263) 
+- Avoid Orion node calling itself for discovery through listening interface [\#261](https://github.com/PegaSysEng/orion/pull/261) 
+- Upgrade jackson [\#258](https://github.com/PegaSysEng/orion/pull/258)
```
Comment (bgravenorst) [inline CHANGELOG.md]: Jackson (uppercase?)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+## 1.2 
+
+- Rename privacyGroupId API to createPrivacyGroupId [\#251](https://github.com/PegaSysEng/orion/pull/251) (thanks to [Puneetha17](https://github.com/Puneetha17))
+- Improve error message when privacy group not found in /send [\#249](https://github.com/PegaSysEng/orion/pull/249) (thanks to [Puneetha17](https://github.com/Puneetha17))
+- Add findPrivacyGroup endpoint [\#247](https://github.com/PegaSysEng/orion/pull/247) (thanks to [Puneetha17](https://github.com/Puneetha17))
+
+## 1.1 
+
+- Rethrow the Exception in SendHandler [\#244](https://github.com/PegaSysEng/orion/pull/244) (thanks to [Puneetha17](https://github.com/Puneetha17))
+- Rethrow the Exception in SendHandler in after getting privacy group. [\#243](https://github.com/PegaSysEng/orion/pull/243) (thanks to [Puneetha17](https://github.com/Puneetha17))
```
Comment (bgravenorst) [inline CHANGELOG.md]: extra "in"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+- Publish source and javadoc to bintray [\#242](https://github.com/PegaSysEng/orion/pull/242) 
+- Upgrade bintray plugin [\#241](https://github.com/PegaSysEng/orion/pull/241) 
+- Support building on JDK 11 and 12 [\#240](https://github.com/PegaSysEng/orion/pull/240) 
+- Keep the generation of privacy group consistent. [\#239](https://github.com/PegaSysEng/orion/pull/239) (thanks to [Puneetha17](https://github.com/Puneetha17))
+- Propagate the delete privacy group changes to all peers [\#238](https://github.com/PegaSysEng/orion/pull/238) (thanks to [Puneetha17](https://github.com/Puneetha17))
+- Push the created privacy group to all the recipients [\#237](https://github.com/PegaSysEng/orion/pull/237) (thanks to [Puneetha17](https://github.com/Puneetha17))
+- Add delete group API in Orion [\#236](https://github.com/PegaSysEng/orion/pull/236) (thanks to [Puneetha17](https://github.com/Puneetha17))
+- Enable send API to accept privacy group Id [\#235](https://github.com/PegaSysEng/orion/pull/235) (thanks to [Puneetha17](https://github.com/Puneetha17))
+- Updating vertx and jackson deps [\#233](https://github.com/PegaSysEng/orion/pull/233)
```
Comment (bgravenorst) [inline CHANGELOG.md]: Is this Vert.x and Jackson dependencies?

File: CHANGELOG.md (hunk 1 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 3 above)
Comment (MadelineMurray) [reply CHANGELOG.md]: Fixed

**Pull request merged**